rmgpy/rmg/pdep.py

```diff
@@ -536,16 +536,45 @@ def remove_disconnected_reactions(self):
         self.Nreac = len(self.reactants)
         self.Nprod = len(self.products)
 
-    def remove_reactions(self,reactionModel,rxns):
+    def remove_reactions(self,reactionModel,rxns=None,prods=None):
         """
         removes a list of reactions from the network and all reactions/products
         left disconnected by removing those reactions
         """
-        for rxn in rxns:
-            self.pathReactions.remove(rxn)
-            
+        if rxns:
```

Comment:
shouldn't this be `if rxns is not None`? Or is there no practical difference?

Reply:
There's no practical difference it won't trigger if rxns = None or rxns = [].